src/tree/subscriptionTreeItem.ts

```diff
@@ -37,6 +49,15 @@ export default class SubscriptionTreeItem extends SubscriptionTreeItemBase imple
         return aksClusterResources.map((aksClusterResource) => new AksClusterTreeItem(this, aksClusterResource));
     }
 
+    public async refreshImpl(): Promise<void> {
```

Comment:
💡🥷🎅🛺  yeah this probably is more sane than this: https://github.com/Azure/vscode-aks-tools/pull/41#discussion_r490639105 .I think it is more favourable to get this in for total experience for the end-users.

At some stage we should really plan to take over the over-all API around this and advancement we could do, just as spitballing ideas, the cross subscription stuff could be the line of thinking we could keep when we take a look at this API for sub tree.

Reply:
Yeah, I think that if we wanted the _cloud_-level refresh button to refresh the subscriptions, we'd need some change to the API, but we'd certainly need to think very carefully about what that change should be.